source/whats-new.txt

```diff
@@ -37,6 +38,29 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.6:
+
+What's New in 5.6
+-----------------
+
+The {+driver-short+} v5.6 release includes the following features:
+
+- The driver now supports Node.js v20.
+
+- The driver can return a cursor as the response to a server command by
+  calling the ``runCursorCommand()`` method. To learn more about this feature,
+  see the `runCursorCommand API documentation <{+api+}/classes/Db.html#command>`__.
```

Comment:
S: link to [this API documentation](https://mongodb.github.io/node-mongodb-native/5.6/classes/Db.html#runCursorCommand) instead of the command type documentation